openstf/tasks/train_model.py

```diff
@@ -52,6 +52,8 @@ def train_model_task(pj: dict, context: TaskContext) -> None:
         pj["hyper_params"]["featureset_name"]
     )
 
+    context.checkpoint("Meta-data")
```

Comment:
Maybe: "Retrieved meta-data" or "Retrieved prediction job"

Reply:
good suggestion, I'll add it